Makefile

```diff
@@ -36,6 +37,13 @@ source-release: clean
 install:
 	$(PYTHON) setup.py install --root $(DESTDIR) $(COMPILE)
 
+pip:
+	$(PYTHON) -m pip --version || $(PYTHON) -c "import os; import sys; import urllib; f = urllib.urlretrieve('https://bootstrap.pypa.io/get-pip.py')[0]; os.system('%s %s' % (sys.executable, f))"
```

Comment:
Please don't (ever) try installing pip without user intervention. That's really intrusive. In case pip is not available the user will see the error. Alternatively you can simply suggest some recommendation by "echo" but not downloading it automatically as users should always favor their distribution channels to `get-pip` script.

Reply:
I just copied these lines from avocado's, so, we would not need this anymore according to the above comment.

Reply:
@ldoktor how is installing pip different from the other requirements?

Reply:
The difference is that dependencies are installed via `pip` which can be set accordingly, while installing pip from external source automatically seems quite intrusive to me. When people have it already and ask `make requirements` to get requirements, sure. No problem. But when they don't have pip at all I think we should only inform the user about possible means to get it.

Reply:
@ldoktor so it **seems** intrusive to you, and you **think** it shouldn't be done.  But in reality, both will download/install software when a user run `make requirements` and doesn't have them installed.

Reply:
... anyway I think we should move the discussion to Avocado as it's not related to this PR (anymore)